Provider/src/FirebirdSql.Data.FirebirdClient/Schema/FbSchema.cs

```diff
@@ -138,5 +160,12 @@ private static void TrimStringFields(DataTable schema)
 		}
 
 		#endregion
+
+		#region Properties
+		/// <summary>
+		/// The major version of the connected Firebird server
+		/// </summary>
+		protected int MajorVersionNumber { get; set; }
```

Comment:
I think the _setter_ can be `private`. Nobody should modify it from outside.

Reply:
agreed